backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/utils/ApplicationHandler.kt

```diff
@@ -128,24 +129,24 @@ class ApplicationHandler(
                 "Physical card must be false if application is already verified"
             }
             val blueCardEntitlement = applicationDetails.blueCardEntitlement
-                ?: throw IllegalArgumentException("Blue card entitlement must be set if application is already verified")
+                ?: throw InvalidJsonException("Blue card entitlement must be set if application is already verified")
 
             val workAtOrganizationsEntitlement = blueCardEntitlement.workAtOrganizationsEntitlement
-                ?: throw IllegalArgumentException("Work at organizations entitlement must be set if application is already verified")
+                ?: throw InvalidJsonException("Work at organizations entitlement must be set if application is already verified")
 
             require(blueCardEntitlement.entitlementType == BlueCardEntitlementType.WORK_AT_ORGANIZATIONS) {
                 "Entitlement type must be WORK_AT_ORGANIZATIONS if application is already verified"
             }
 
             val organizations = workAtOrganizationsEntitlement.list
-            require(!organizations.isNullOrEmpty()) {
+            require(organizations.isNotEmpty()) {
                 "Work at organizations list cannot be empty if application is already verified"
             }
             require(organizations.all { it.organization.category.shortText == "Sport" }) {
                 "All organizations must be of category Sport if application is already verified"
             }
         } catch (e: IllegalArgumentException) {
-            throw BadRequestResponse(e.message!!)
+            throw InvalidJsonException(e.message!!)
```

Comment:
Replaced with InvalidJsonException to stick to the common error format.
(we have a custom handler for GraphQLBaseExceptions to convert them to a specific response format)